content/en/developers/metrics/_index.md

```diff
@@ -17,34 +18,40 @@ further_reading:
   text: "Official and Community created API and DogStatsD client libraries"
 ---
 
-A Datadog metric is defined by the properties below. Refer to the [Metrics Introduction documentation][1] to learn how to graph metrics within Datadog.
+## Overview
+
+If a metric is not submitted from one of the [350+ Datadog integrations][1] it's considered a custom metric<sup>(1)</sup>.
```

Comment:
Can we add something about what a custom metric is, rather than strictly what it is not, or a link to a neat blog? Custom metrics measure often telemetry about an application, Examples of custom metrics include critical business telemetry like time on page, or other operational data about the behavior of a custom application or something?